BitwardenShared/UI/Platform/Application/Support/Images.xcassets/Images/empty_vault_items.imageset/Contents.json

```diff
@@ -0,0 +1,25 @@
+{
+  "images" : [
+    {
+      "filename" : "empty_vault_items_light.pdf",
```

Comment:
❓ Does it make sense to name this the same as the illustrations name in Figma (`items`)? There's pros/cons to both, but I like the fact that it matches Figma and we could then import all illustrations up front and not have to do the mapping of changing the name to where it's used.

Reply:
yeah, i can do that!